### ISSUE TITLE
Proposed force touch options for iOS devices

### ISSUE THREAD

**Issue by srirambv** [brave/browser-ios#814]
Current 
![image](https://cloud.githubusercontent.com/assets/17010094/25065866/068f21de-220f-11e7-9806-4bc411e93ee3.png)

Proposed 
![image](https://cloud.githubusercontent.com/assets/17010094/25065918/1e65def0-2210-11e7-9a0b-84cbe97e56f5.png)

cc: @jamesmudgett @bradleyrichter @jhreis

**Comment by jhreis** [comment]
Yes, definitely some improvements. Personally don't think we need any Sync things, as it is such an infrequent thing to use. 

Widget support would be super nice to have. Not sure how flexible widgets are, so will investigate that a bit more.

I think we also should support something similar to FF and Chrome with being able to open a link if it is already in the clipboard. (copy link, force touch on one of those, and there is an option to open the link).

**Comment by jhreis** [comment]
Related: #736

**Comment by bradleyrichter** [comment]
Agree that Sync is not needed and should be "set it and forget it" when we get it working correctly. 

My suggestion would be:

- New Tab
- New Private Tab
- Search using clipboard (or Visit URL on Clipboard if known)
- Show Bookmarks
- Close All Tabs (saves memory and useful when handing browser to another person)
--------------------
- Share Brave (I'm not a fan but it seems like Apple stuffs this there)
